Paper.tex

```diff
@@ -321,7 +321,7 @@ \subsection{The Transaction} \label{subsec:transaction}
 \mathbb{B}_{0} & \text{otherwise}\end{cases}
 \end{equation}
 
-\subsection{The Block} \linkdest{block}{}
+\subsection{The Block}\linkdest{block}\label{subsec:The_Block}
```

Comment:
How is this parsed?  The second argument of `\linkdest` is `\label`?

Reply:
Not quite. `linkdest` will just refer to the line that it's on. But when you have `\hyperlink{block}{\ref{subsec:The_Block}}`, it will display:
![screenshot from 2018-02-20 13-12-22](https://user-images.githubusercontent.com/16969914/36404611-c356f74a-163f-11e8-9327-ba524a573535.png)
Try building it!

Reply:
Then the link will jump to:
![screenshot from 2018-02-20 13-13-40](https://user-images.githubusercontent.com/16969914/36404622-e6225e40-163f-11e8-8423-aee00707658d.png)

Reply:
I built it before writing my comment, of course.  The build was a success, but LaTeX sometimes does nonsense.

Reply:
Ah, `\linkdest` takes only one argument.  I missed that.